packages/bodiless-core/src/Store/StoreItem.ts

```diff
@@ -15,7 +15,7 @@
 import path from 'path';
```

Comment:
Is it worth creating a static version of the storeItem which removes all the logic around posting to the backend and debouncing?  Or are the savings too small...

Reply:
Done